.github/workflows/action.yml

```diff
@@ -1,7 +1,7 @@
 ---
 # yamllint disable-line rule:truthy
 name: 'Auto Assign'
-on: pull_request
```

Comment:
should it be for both `pull_request` and `pull_request_target`?

Reply:
according to the docs it will work for both purposes since it takes the base repository https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request_target